src/queue.c

```diff
@@ -6490,7 +6490,9 @@ _dispatch_runloop_queue_get_handle(dispatch_lane_t dq)
 #elif defined(__linux__)
 	// decode: 0 is a valid fd, so offset by 1 to distinguish from NULL
 	return ((dispatch_runloop_handle_t)(uintptr_t)dq->do_ctxt) - 1;
-#elif defined(__unix__) && !defined(__linux__)
+#elif defined(__FreeBSD__)
+	return ((dispatch_runloop_handle_t)(uintptr_t)dq->do_ctxt);
```

Comment:
Do we need this? It looks like it's the same as the `defined(__unix__)` case below, which should capture on FreeBSD.